spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe "Contestants Index Page", type: :feature do
+  describe "When I visit the contestants index page ('/contestants')" do
+    before :each do
+      @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+  
+      @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+      @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+      @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+      @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+  
+      @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+      @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+  
+      ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+      ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+      ContestantProject.create(contestant_id: @gretchen.id, project_id: @boardfit.id)
+      ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+      ContestantProject.create(contestant_id: @erin.id, project_id: @boardfit.id)
+  
+      visit "/contestants"
+    end
+
+    it 'has a list of names of all the contestants' do
+      expect(page).to have_content(@jay.name)
+      expect(page).to have_content(@gretchen.name)
+      expect(page).to have_content(@kentaro.name)
+      expect(page).to have_content(@erin.name)
+    end
+
+    it 'under each contestants name I see a list of the projects (names) that theyve been on' do
+      expect(page).to have_content(@jay.project_names)
```

Comment:
You could make this test more robust by using within blocks (and adding appropriate CSS class/id info in your view) so that you can be more explicit about where the project names are listed (i.e. News Chic is listed for Jay but *not* for Kentaro)